src/Disks/ReadIndirectBufferFromRemoteFS.h

```diff
@@ -5,43 +5,34 @@
 #endif
 
 #include <IO/ReadBufferFromFile.h>
+#include <IO/ReadBufferFromRemoteFS.h>
 #include <Disks/IDiskRemote.h>
 #include <utility>
 
 
 namespace DB
 {
 
-/// Reads data from S3/HDFS using stored paths in metadata.
-template <typename T>
+/// Reads data from S3/HDFS/Web using stored paths in metadata.
 class ReadIndirectBufferFromRemoteFS : public ReadBufferFromFileBase
```

Comment:
So maybe choose better name?

Reply:
But adding `indirect` to name is not only for remote fs, there is also `readIndirectBuffer` in `DiskMemory`. So it is more consistent with these names.